gettsim/data/e_st_abzuege.yaml

```diff
@@ -123,6 +126,8 @@ vorwegab:
     2019-01-01:
       value: 300
       note:
+    2020-01-01:
+      value: 0
```

Comment:
There is no note because these are values from a table given in §10. And you are right, the last value is 300 and not zero. One could handle this via a year switch, but it would actually be something for #43.